app/config/locale/translations/be.json

```diff
@@ -3,30 +3,30 @@
     "settings.locale": "be",
     "settings.direction": "ltr",
     "emails.sender": "Каманда %s",
-    "emails.verification.subject": "",
-    "emails.verification.hello": "",
-    "emails.verification.body": "",
-    "emails.verification.footer": "",
-    "emails.verification.thanks": "",
-    "emails.verification.signature": "",
-    "emails.magicSession.subject": "",
-    "emails.magicSession.hello": "",
-    "emails.magicSession.body": "",
-    "emails.magicSession.footer": "",
-    "emails.magicSession.thanks": "",
-    "emails.magicSession.signature": "",
-    "emails.recovery.subject": "",
-    "emails.recovery.hello": "",
-    "emails.recovery.body": "",
-    "emails.recovery.footer": "",
-    "emails.recovery.thanks": "",
-    "emails.recovery.signature": "",
-    "emails.invitation.subject": "",
-    "emails.invitation.hello": "",
-    "emails.invitation.body": "",
-    "emails.invitation.footer": "",
-    "emails.invitation.thanks": "",
-    "emails.invitation.signature": "",
+    "emails.verification.subject": "Верыфікацыя акаўнта",
+    "emails.verification.hello": "Прывітанне {{name}}",
+    "emails.verification.body": "Перайдзіце па гэтай спасылцы, каб пацвердзіць свой адрас электроннай пошты",
+    "emails.verification.footer": "Калі вы не запытвалі пацвярджэнне гэтага адрасу, праігнаруйце гэтае паведамленне.",
+    "emails.verification.thanks": "Дзякуем",
+    "emails.verification.signature": "каманда {{project}}",
+    "emails.magicSession.subject": "Лагін",
+    "emails.magicSession.hello": "Прывітанне,",
+    "emails.magicSession.body": "Перайдзіце па спасылцы, каб увайсці.",
+    "emails.magicSession.footer": "Калі вы не прасілі ўвайсці, выкарыстоўваючы гэты адрас электроннай пошты, праігнаруйце гэтае паведамленне.",
+    "emails.magicSession.thanks": "Дзякуем",
+    "emails.magicSession.signature": "каманда {{project}}",
+    "emails.recovery.subject": "Скід пароля",
+    "emails.recovery.hello": "Прывітанне, {{name}}",
+    "emails.recovery.body": "Перайдзіце па гэтай спасылцы, каб скінуць пароль для праекта {{project}}.",
+    "emails.recovery.footer": "Калі вы не прасілі скінуць пароль, вы можаце праігнараваць гэта паведамленне.",
+    "emails.recovery.thanks": "Дзякуем",
+    "emails.recovery.signature": "каманда {{project}}",
+    "emails.invitation.subject": "Запрошення до Команди %s у %s",
+    "emails.invitation.hello": "Прывітанне",
+    "emails.invitation.body": "Гэта паведамленне было адпраўлена вам, таму што <b>{{owner}}</b> хацеў запрасіць вас стаць членам каманды <b>{{team}}</b> у {{project}}.",
```

Comment:
Using `<b>` tags is a great addition, but it may not be recommended since our baseline [English translation](https://github.com/appwrite/appwrite/blob/master/app/config/locale/translations/en.json#L26) and others do not use bold lettering. We can do this in the future but let's keep it simple for the sake of this PR.

Reply:
done!